shell/components/cards/ItemCard.vue

```diff
@@ -0,0 +1,366 @@
+              class="item-card-pill"
+              :aria-label="labelText(pill.tooltip) || labelText(pill.label)"
+              data-testid="item-card-pill"
+            >
+              {{ labelText(pill.label) }}
+            </div>
+          </slot>
+        </div>
+      </template>
+
+      <div class="item-card-body-details">
+        <div :class="['item-card-header', variant]">
+          <template v-if="variant === 'small'">
+            <slot name="item-card-image">
+              <div
+                v-if="image"
+                :class="['item-card-image', variant]"
+                data-testid="item-card-image"
+              >
+                <LazyImage
+                  :src="image.src"
+                  :alt="labelText(image?.alt)"
+                />
+              </div>
+            </slot>
+          </template>
+          <slot name="item-card-header-title">
+            <h3
+              v-clean-tooltip="labelText(header.title)"
+              :class="['item-card-header-title', variant]"
+              :aria-label="labelText(header.title)"
+              data-testid="item-card-header-title"
+            >
+              {{ labelText(header.title) }}
+            </h3>
+          </slot>
+          <div
+            v-if="header.statuses"
+            class="item-card-header-statuses"
+          >
+            <div
+              v-for="(status, i) in visibleStatuses"
+              :key="i"
+              class="item-card-header-statuses-status"
+              :aria-label="labelText(status.tooltip)"
+              data-testid="item-card-header-statuses-status"
+            >
+              <i
+                v-clean-tooltip="labelText(status.tooltip)"
+                :class="['icon', status.icon, status.color]"
+                :style="{color: status.customColor}"
+                :data-testid="`item-card-header-status-${i}`"
+              />
+            </div>
+          </div>
+
+          <template v-if="$slots['item-card-actions']">
+            <div class="item-card-header-action-menu no-card-click">
+              <slot name="item-card-actions" />
+            </div>
+          </template>
+        </div>
+
+        <slot name="item-card-sub-header" />
+
+        <template v-if="$slots['item-card-content']">
+          <slot name="item-card-content">
+            <div
+              class="item-card-content"
+              data-testid="item-card-content"
+            />
+          </slot>
+        </template>
+        <template v-else-if="content">
+          <div
+            class="item-card-content"
+            aria-label="Card content"
+            data-testid="item-card-content"
+          >
+            <p>{{ labelText(content) }}</p>
+          </div>
+        </template>
+
+        <slot name="item-card-footer" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped lang="scss">
+.item-card {
+  display: flex;
+  min-width: 420px;
```

Comment:
I think we can remove the `min-width`, it works the same without this property + it's causing same layout issue in Fleet